torch/csrc/jit/codegen/cuda/ir_iostream.cpp

```diff
@@ -40,7 +40,8 @@ void IRPrinter::handle(const Expr* e) {
 void IRPrinter::printHeader(
     Fusion* fusion,
     const std::string& kernel_name_,
-    const std::vector<Val*>& global_buffers) {
+    const std::vector<Val*>& global_buffers,
+    const bool hasDynamicSmem) {
```

Comment:
no `const` on arguments please (note this is different than passing a const reference)